tock/tock/templates/employees/user_recent_tocks.html

```diff
@@ -0,0 +1,28 @@
+<h2>Recent Tocks</h2>
+<table>
+  <thead>
+    <th>Project</th>
+    {% for tc in recent_timecards %}
+    <th><a
+            href={% url 'reports:ReportingPeriodUserDetailView'
+                    reporting_period=tc.reporting_period.start_date
+                    username=tc.user.username %}
+        >
+            {{tc.reporting_period.start_date}}
+        </a>
+    </th>
+
+    {% endfor %}
+  </thead>
+  <tbody>
+
+  </tbody>
```

Comment:
I think you closed your tbody prematurely.

Reply:
Ah thanks!

Amended w/ correction